src/pages/workspaces/workspace/notebooks/TerminalLauncher.js

```diff
@@ -101,10 +101,18 @@ const TerminalLauncher = _.flow(
       })
     } else {
       return div({ style: { padding: '2rem' } }, [
-        spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
-        (clusterStatus === 'Creating' || !cluster) ?
-          'Creating runtime environment. You can navigate away and return in 5-10 minutes.' :
-          'Starting runtime environment, this may take up to 2 minutes.'
+        (clusterStatus !== 'Stopped') && spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
+        (() => {
+          if (clusterStatus === 'Creating' || !cluster) {
```

Comment:
nit/tol: for consistency would it make sense to have all the status reports call the space the same name, either 'notebook runtime environment' or 'runtime environment'?